xyz-models/src/main/java/com/here/xyz/models/hub/Space.java

```diff
@@ -317,13 +325,31 @@ public Extension getExtension() {
 
   public void setExtension(final Extension extension) {
     this.extension = extension;
+    if (extension != null) {
```

Comment:
This could be done as part of the "model to item transformation" inside `DynamoSpaceConfigClient#storeSpace(Marker marker, Space space, Space originalSpace)`

Reply:
ok